packages/angular/src/builders/utilities/webpack.ts

```diff
@@ -27,7 +28,7 @@ export async function mergeCustomWebpackConfig(
 export function resolveCustomWebpackConfig(path: string, tsConfig: string) {
   tsNodeRegister(path, tsConfig);
 
-  const customWebpackConfig = require(path);
+  const customWebpackConfig = __importDefault(require(path));
```

Comment:
Hey :) Can you explain why you're using this from `tslib`? 

We don't want to directly use tslib functions in this code, so if this can be done in a different manner, I would suggest to do that.